src/styles/ToggleTheme.css

```diff
@@ -0,0 +1,41 @@
+input[type=checkbox]{
```

Comment:
The indentation is all out here, should be moved to 2 spaces as with the rest of the project